src/main/java/net/sf/jabref/gui/actions/BrowseAction.java

```diff
@@ -38,33 +38,35 @@
     private final JTextField comp;
     private final boolean dir;
     private final JComponent focusTarget;
+    private final String extension;
 
     public static BrowseAction buildForDir(JFrame frame, JTextField tc) {
-        return new BrowseAction(frame, tc, true, null);
+        return new BrowseAction(frame, tc, true, null, Globals.NONE);
     }
 
     public static BrowseAction buildForDir(JTextField tc) {
-        return new BrowseAction(null, tc, true, null);
+        return new BrowseAction(null, tc, true, null, Globals.NONE);
     }
 
     public static BrowseAction buildForFile(JTextField tc) {
-        return new BrowseAction(null, tc, false, null);
+        return new BrowseAction(null, tc, false, null, Globals.NONE);
```

Comment:
Yes, you are right. Interestingly the Globals.NONE was used before. I will look into that this evening.